### DISCUSSION TITLE
Multiple test categories

### DISCUSSION THREAD

**Discussion by ledjon-behluli** [thomhurst/TUnit#1204] [Q&A] [answered]
if i have something like this:

```
[Test, Category("Offender")]
public Task MyTest(){}
```

I can run it like this: dotnet run -c Release -v minimal --treenode-filter /*/*/*/*[Category=Offender]

How would the cmd line look like if I wanted to run tests which have multiple categories like this:

```
[Test, Category("Offender"), Category("BVT")]
public Task MyTest(){}
```

much thanks

**Comment by thomhurst** [comment, marked as answer]
You want to run tests that have two categories?

That'd be:

`dotnet run -c Release -v minimal --treenode-filter /*/*/*/*[(Category=Offender)&(Category=BVT)]`

**Reply by ledjon-behluli** [reply]
Lovely, thanks! Would be nice to add it to the docs
